src/main/scala/is/hail/expr/Type.scala

```diff
@@ -730,7 +743,7 @@ case class TDict(keyType: Type, valueType: Type) extends TContainer {
 case object TGenotype extends ComplexType {
   override def toString = "Genotype"
 
-  override lazy val fundamentalType: TStruct = TStruct(
+  override lazy val representation: TStruct = TStruct(
```

Comment:
`representation` is used to build the Spark schema when converting to a data frame, e.g. `kt.toDF`, but there we need to keep the distinction between Binary and String.